src/api/app/assets/stylesheets/webui2/cm2/suse.scss

```diff
@@ -1,61 +1,69 @@
-.cm-s-bootstrap.CodeMirror {
-  font-family: $font-family-monospace, monospace;
-  height: auto;
-  background: $card-bg;
-  color: $body-color;
-  line-height: 1.2;
-}
+.cm-s-bootstrap {
+  &.CodeMirror {
+    font-family: $font-family-monospace, monospace;
+    height: auto;
+    background: $card-bg;
+    color: $body-color;
+    line-height: 1.2;
+  }
 
-.cm-s-bootstrap .cm-header {color: $blue;}
-.cm-s-bootstrap .cm-quote {color: $green;}
-
-.cm-s-bootstrap .cm-keyword {color: $pink;}
-.cm-s-bootstrap .cm-atom {color: $indigo;}
-.cm-s-bootstrap .cm-number {color: $teal;}
-.cm-s-bootstrap .cm-def {color: $blue;}
-.cm-s-bootstrap .cm-variable,
-.cm-s-bootstrap .cm-punctuation,
-.cm-s-bootstrap .cm-property,
-.cm-s-bootstrap .cm-operator {color: $body-color;}
-.cm-s-bootstrap .cm-variable-2 {color: $blue;}
-.cm-s-bootstrap .cm-variable-3, .cm-s-bootstrap .cm-type {color: $teal;}
-.cm-s-bootstrap .cm-comment {color: $orange;}
-.cm-s-bootstrap .cm-string {color: $danger;}
-.cm-s-bootstrap .cm-string-2 {color: $orange;}
-.cm-s-bootstrap .cm-meta {color: $text-muted;}
-.cm-s-bootstrap .cm-qualifier {color: $text-muted;}
-.cm-s-bootstrap .cm-builtin {color: $purple;}
-.cm-s-bootstrap .cm-bracket {color: $warning;}
-.cm-s-bootstrap .cm-tag {color: $success;}
-.cm-s-bootstrap .cm-attribute {color: $blue;}
-.cm-s-bootstrap .cm-hr {color: $secondary;}
-.cm-s-bootstrap .cm-link {color: $link-color;}
-
-.cm-s-bootstrap .cm-error {color: $danger;}
-
-.cm-s-bootstrap .cm-negative {
-  color: $danger;
-  background: rgba($danger, 0.1);
-}
+  .cm-header {color: $blue;}
```

Comment:
Some of these can be grouped and with a line to space it is out, it will be more readable.

Reply:
but you realize I just removed the duplication? And I disagree with you - they are grouped by mode specifics and editor specifcs.

Reply:
Yes, I've seen that you use Sass like it should (before it was just CSS with some Sass variables).

Good point regarding the modes.